### PR TITLE
Fixing broken inline links

### PR DESCRIPTION
**Pull request by koshy-thomas**

### CODE REVIEW

File: docs/c8ql/functions/search.md [hunk 1]
```diff
@@ -187,7 +187,7 @@ Match documents where the attribute at **path** is present _and_ was indexed by
 
 - **path** (attribute path expression): the attribute to test in the document
 - **type** (string): string literal `"analyzer"`
-- **analyzer** (string, _optional_): name of an [Analyzer](../../search/analyzers.md). Uses the Analyzer of a wrapping `ANALYZER()` call if not specified or defaults to `"identity"`
+- **analyzer** (string, _optional_): name of an [Analyzer](../../search/Analyzers.md). Uses the Analyzer of a wrapping `ANALYZER()` call if not specified or defaults to `"identity"`
```
Comment (diana-macrometa) [inline docs/c8ql/functions/search.md]: All filenames are lowercase. Please back this one out.

Comment (diana-macrometa) [reply docs/c8ql/functions/search.md]: And all others that change lowercase to uppercase.

**Review by diana-macrometa — CHANGES_REQUESTED** [review]
Thanks so much for doing this! Just need to make sure that all filenames are lowercase, not uppercase.

File: docs/c8ql/functions/search.md (hunk 1 above)
Comment (koshy-thomas) [reply docs/c8ql/functions/search.md]: In that case I would need to change the filename here since the filename is in Uppercase here
image

Comment (diana-macrometa) [reply docs/c8ql/functions/search.md]: Try updating from master. Those uppercase files were changed to lowercase last week.

Comment (koshy-thomas) [reply docs/c8ql/functions/search.md]: image

Comment (koshy-thomas) [reply docs/c8ql/functions/search.md]: Unfortunately I don't see that change in master too

Comment (koshy-thomas) [reply docs/c8ql/functions/search.md]: I will just go ahead and change those files to lowercase. I hope that's okay?

Comment (diana-macrometa) [reply docs/c8ql/functions/search.md]: Yes, please do.

@elof, is this the cache refresh issue you mentioned? The files are lowercase on my local machine, which is current with master.

Comment (elof) [reply docs/c8ql/functions/search.md]: @diana-macrometa it shouldn't be a cache issue if you are running it locally cc @koshy-thomas

Comment (koshy-thomas) [reply docs/c8ql/functions/search.md]: Hello I have updated the filenames please review again and let me know if things are fine now

Comment (koshy-thomas) [reply docs/c8ql/functions/search.md]: cc @elof @diana-macrometa

Comment (diana-macrometa) [reply docs/c8ql/functions/search.md]: @koshy-thomas, if you click the circling arrows next to someone's name at the top of the right column, then the system will send out the review request for you. :)

**Review by diana-macrometa — APPROVED** [review]
LGTM. Thanks, Koshy!

**Pull request merged**